components/teams/TeamMember.tsx

```
@@ -0,0 +1,96 @@
+                </Avatar>
+                <span>{member.name}</span>
+              </div>
+            </TableCell>
+            <TableCell>{member.role}</TableCell>
+            <TableCell>{member.joinedOn}</TableCell>
+            <TableCell>{member.tasksAssigned}</TableCell>
+            {isAdmin && (
+              <TableCell>
+                <DropdownMenu>
+                  <DropdownMenuTrigger asChild>
+                    <button className="rounded-full p-2 hover:bg-gray-100">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)